src/routes/newsletter/+page.svelte

```diff
@@ -2,7 +2,7 @@
   <title>Subscribe to Blend</title>
 </svelte:head>
 
-<form method="post" action="https://communications.blendreading.com/subscription/form" class="listmonk-form">
+<form method="post" action="https://communications.blendreading.com/subscription/form" class="listmonk-form" on:submit={ () => gtag('event', 'subscribe_newsletter') }>
```

Comment:
This will run into the same race condition as the `a` tag, I think. But I'm pretty confident that the gtag event will fire off before the listmonk server responds. If we want to be sure we need to handle it though